sql/catalyst/src/test/scala/org/apache/spark/sql/util/DateFormatterSuite.scala

```diff
@@ -89,4 +89,10 @@ class DateFormatterSuite extends SparkFunSuite with SQLHelper {
       }
     }
   }
+
+  test("parsing date with locale") {
```

Comment:
all other tests in this suite have locale, is this test really to test locale?

Reply:
It actually test how a date without explicit day is parsed. I will correct test title.